### DISCUSSION TITLE
Rapi command for testing the "StuckRelay" or other initial test.

### DISCUSSION THREAD

**Discussion by DylanVautrin** [OpenEVSE/open_evse#27] [Q&A] [answered]
Hello,
I'm working on a project using OpenEVSE and I would like to create a test sequence. I know that the OpenEVSE board make some test when it boots, but is there the rapi command to make it "whenever" I want ? Like to trigger the test for the relay, for example. 
I looked for it, but I can't find a way to make this, someone can help me please ?
Thanks

**Comment by chris1howell** [comment, marked as answer]
OpenEVSE runs tests on boot, continuously and prior to closing the relay based on UL requirements. All tests are automatic, there is no way to trigger a specific test.

-Stuck Relay is running continuously anytime the relay should be open.
-Ground Monitoring is running continuously anytime the relay is closed "while charging" a hardware update will be coming soon to ground test "always" per an recent UL update.
-GFCI Self test is run on boot and any time prior to closing the relay 

A flow chart of all the safety tests is here:
https://openev.freshdesk.com/helpdesk/attachments/6061015834

Here is a Knowledge Base article:
https://openev.freshdesk.com/en/support/solutions/articles/6000113537-openevse-safety-features

**Reply by DylanVautrin** [reply]
OK, thanks for support.
